README.md

```diff
@@ -17,53 +17,139 @@ The Sentry extension renders a `View logs in Sentry` next to error throwing stat
 
 ## Language support
 
-To work, the Sentry Sourcegraph extension must know how to recognize instances of error handling and/or exception throwing for each language. The first version will support:
+The Sentry Sourcegraph extension uses common error handling and/or exception throwing patterns specific to each language to identfy Sentry relevant lines of code. The following languages are currently supported:
 
 - TypeScript
 - Go
 - JavaScript
 - Python
 - Java
 
-## Setup
+## Basic Setup
 
-Set the following configurations in your settings:
+In your Sourcegraph settings (user, organization, or global), add the following configuration:
```

Comment:
I would mention that having this be set once by the site admin in org or global settings is the recommended way, so that individual org users do not have to configure this individually.